fonts/generate_font_metadata.py

```diff
@@ -45,7 +45,7 @@ def validateGlyph(glyph):
             report += '- DUPLICATE UNICODE POINT'
 
         if report != '':
-            print('%s (%#0.2x) issues: %s' % (g.glyphname, glyph.unicode, report), log)
```

Comment:
F strings are used elsewhere — why not here?

Reply:
This file originally comes from https://github.com/w3c/smufl repo. We removed all the log file stuff and this one was overlooked. I'm still wondering if it should be in line with the upstream version and if we should push further fixes/improvements there?